conf/base.config

```diff
@@ -27,24 +27,24 @@ process {
     // TODO nf-core: Customise requirements for specific processes.
     // See https://www.nextflow.io/docs/latest/config.html#config-process-selectors
     withLabel:process_single {
-        cpus   = { 1                   }
-        memory = { 6.GB * task.attempt }
-        time   = { 4.h  * task.attempt }
+        cpus   = { 20      * task.attempt }
+        memory = { 80.GB   * task.attempt }
+        time   = { 24.h    * task.attempt }
     }
     withLabel:process_low {
-        cpus   = { 2     * task.attempt }
-        memory = { 12.GB * task.attempt }
-        time   = { 4.h   * task.attempt }
+        cpus   = { 20      * task.attempt }
+        memory = { 80.GB   * task.attempt }
+        time   = { 24.h    * task.attempt }
     }
     withLabel:process_medium {
-        cpus   = { 6     * task.attempt }
-        memory = { 36.GB * task.attempt }
-        time   = { 8.h   * task.attempt }
+        cpus   = { 20      * task.attempt }
+        memory = { 80.GB   * task.attempt }
+        time   = { 24.h    * task.attempt }
     }
     withLabel:process_high {
-        cpus   = { 12    * task.attempt }
-        memory = { 72.GB * task.attempt }
-        time   = { 16.h  * task.attempt }
+        cpus   = { 20    * task.attempt }
+        memory = { 80.GB * task.attempt }
+        time   = { 24.h  * task.attempt }
     }
```

Comment:
The base configs are too resource intesive. These are supposed to be sensible defaults, which can be overridden by the user.